layout: migrate frontpage elements to semantic-ui

mvidalgarcia opened the pull request:
- layout: adapt header to semantic-ui
- layout: adapt footer to semantic-ui
- layout: adapt main.html to semantic-ui
- layout: adapt LVA section to semantic-ui
- layout: adapt news section to semantic-ui

closes #3006



https://user-images.githubusercontent.com/8863238/105725170-eaf22380-5f28-11eb-9508-c8d21b328f88.mov

audrium commented:
Very nice work! :rocket:  Just couple of minor observations:
- I noticed that header is quite huge now compared to what we have in production. `77px` compared to `65px`
- Also regarding the header:
    - if I click "About" multiple times fast, it selects the text
    - On mobile view "About" section is hidden
    - If I open "About" dropdown and select for example CMS on the side (when text is not underlined) it gets rendered in bold and does not redirect:
![Screenshot from 2021-01-26 16-24-38](https://user-images.githubusercontent.com/4990025/105865308-16d9db80-5ff3-11eb-90aa-56c78e61ed21.png)
- Search bar lost nice border colour when focused. On Chrome it's even rendered in black for me:
![Screenshot from 2021-01-26 15-56-40](https://user-images.githubusercontent.com/4990025/105862216-d593fc80-5fef-11eb-9b76-876f4b50b943.png) (FF renders it in blue)
- Also "Start typing..." placeholder is very near the search bar border. Is it possible to add some padding?
- In "Get Started" section I noticed some extra space between `header` and `body`. By removing `margin-bottom: 2em;` from `header` it worked in the same way as in production. This value comes from `invenio` and was placed in `invenio_theme/theme/globals/site.overrides`
- "News & Updates" section looks way better now :rocket: But for me hovering on CMS label makes it very light:
![Screenshot from 2021-01-26 16-17-49](https://user-images.githubusercontent.com/4990025/105864361-2278d280-5ff2-11eb-8bc9-b7d498178c10.png)

mvidalgarcia commented:
@audrium I've addressed all the issues you mentioned. Thanks for the detailed report!

The mobile view is quite broken ATM, as we get many default styles from Invenio-Theme, but probably we can create a different issue to tackle responsiveness,  what do you think?

audrium commented:
Just retested, everything works nicely! Turbo :1st_place_medal: 
Totally agree that we shouldn't worry to much about the mobile view now and we can address it later with different issue
